src/app/form-styles/elem-list/elem-list.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { select, Store } from '@ngrx/store';
+import { Observable } from 'rxjs';
+
+import { selectParamsElemForId } from '../../reducers/elemStyles/elemStyles.selectors';
+import { SetStyleElemAction, SetParamElemAction } from '../../reducers/elemStyles/elemStyles.actions';
+import { ListElemStyleState } from 'src/app/data/data';
+
+@Component({
+  selector: 'app-elem-list',
+  templateUrl: './elem-list.component.html',
+  styleUrls: ['./elem-list.component.scss']
+})
+export class ElemListComponent {
+
+  @Input() targetElemId: number;
+
+  public listParamsElem$: Observable<string>;
+
+  listStyles: any = [];
+
+  arrNamesStyles = [
+    "width",
+    "height",
+    "borderWidth",
+    "color", ,
+    "borderStyle",
+    "fontSize",
+    "fontWeight",
+    "backgroundColor",
+    "borderColor"
+  ];
+
+  constructor(private store$: Store<ListElemStyleState>) { }
+
+  ngOnChanges(changes: any) {
+    if (changes['targetElemId']) {
+      this.targetElemId = changes['targetElemId'].currentValue;
+    }
+    this.listParamsElem$ = this.store$.pipe(select(selectParamsElemForId, { id: this.targetElemId }))
+    this.listParamsElem$.subscribe(subs => this.listStyles = subs);
+  }
+
+  onChange(event: any): void {
+    let id = Number(event.target.parentNode.parentNode.id);
+    let name = event.target.name;
+    let value = event.target.value;
```

Comment:
use  const instead of let if the value will not change
https://stackoverflow.com/questions/41086633/why-most-of-the-time-should-i-use-const-instead-of-let-in-javascript
```suggestion
    const id = Number(event.target.parentNode.parentNode.id);
    const name = event.target.name;
    const value = event.target.value;
```